src/controllers/aer_controller.hpp

```diff
@@ -1007,7 +997,7 @@ Result Controller::execute(std::vector<Circuit> &circuits,
     auto timer_stop = myclock_t::now();
     auto time_taken =
         std::chrono::duration<double>(timer_stop - timer_start).count();
-    result.metadata.add(time_taken, "time_taken");
+    result.metadata.add(time_taken, "time_taken_execute");
   }
```

Comment:
This feels like it could be a change to the public API?  If so, it maybe needs a release note.

Reply:
This was just overriden by the outer time taken call before so was never seen by users. Now it will be a new entry in `result.metadata` so you can actually see it

Reply:
ok, sounds good.